File: GeeksForGeeks-Problems/JAVASCRIPT/Functional Programming/Decimal Number to Binary Number.js
// User function Template for javascript

class Solution {
  toBinary(n) {
    // write your code here
    let ans = "";
    if (n == 0) return "0";
    while (n > 0) {
      let rem = n % 2;
      ans += rem;
      n = Math.floor(n / 2);
    }
    return ans.split("").reverse().join("");
  }
}

File: GeeksForGeeks-Problems/CPP/String/Decimal Number to Binary Number.cpp
// User function Template for C++

string toBinary(int n)
{
    // Your code here
    string ans = "";
    while (n > 0)
    {
        int rem = n % 2;
        ans += (rem + '0');
        n /= 2;
    }
    reverse(ans.begin(), ans.end());
    return ans;
}